Modules/mathmodule.c

```diff
@@ -2070,11 +2070,6 @@ math_trunc(PyObject *module, PyObject *x)
         return PyFloat_Type.tp_as_number->nb_int(x);
     }
 
-    if (!_PyType_IsReady(Py_TYPE(x))) {
```

Comment:
Please verify with @serhiy-storchaka that this is unused. Thank you.

Reply:
Thanks Carol, I have reached out to Serhiy on X.